src/components/ProfileImageUploader.js

```diff
@@ -0,0 +1,178 @@
+      }
+    }
+    setIsUploading(false)
+    setShow(false)
+    window.location.reload()
+  }
+
+  const { getRootProps, getInputProps } = useDropzone({
+    multiple: false,
+    accept: 'image/jpeg, image/png, image/gif',
+    minSize: 0,
+    maxSize: 153600,
+    onDrop: ([newFile], [rejectedFile]) => {
+      try {
+        setFile(
+          Object.assign(newFile, {
+            preview: URL.createObjectURL(newFile)
+          })
+        )
+        setError(null)
+      } catch (err) {
+        const fileErrors = rejectedFile.errors.map((error) => error.code)
+        if (fileErrors.indexOf('file-too-large') !== -1) {
+          setError('Please upload a file smaller than 150 kB')
+        }
+        if (fileErrors.indexOf('file-invalid-type') !== -1) {
+          setError(
+            'Please upload a file with .png, .jpg/.jpeg or .gif extensions'
+          )
+        }
+      }
+    }
+  })
+
+  useEffect(
+    () => () => {
+      URL.revokeObjectURL(file?.preview)
+    },
+    [file]
+  )
+
+  return isUploading ? (
+    <Modal isActive={true} showButtons={false}>
+      <LoadingIndicator />
+    </Modal>
+  ) : (
+    <Modal
+      isActive={show}
+      handleClose={() => setShow(false)}
+      onCancel={clearModal}
+      okText={'Save'}
+      onOk={upload}
+      showButtons
+    >
+      <section className='container'>
+        {!removedProfilePic && (
+          <aside className='preview-container'>
+            <span
+              className='remove-img-icon'
+              onClick={() => {
+                setRemovedProfilePic(true)
+                setFile(null)
+              }}
+            >
+              <i className='eos-icons'>delete</i>
+            </span>
+            <img
+              src={profilePicURL}
+              className='user-profile-avatar-img'
+              alt='preview'
+            />
+          </aside>
+        )}
+        {removedProfilePic &&
+          (!file ? (
+            <div {...getRootProps({ className: 'dropzone' })}>
+              <input {...getInputProps()} />
+              <p>
+                Drag 'n' drop your image here, or click to select one. For best
+                results upload images with 1:1 aspect ratio. Maximum allowed
+                file-size is 150 kB.
+              </p>
+            </div>
+          ) : (
+            <aside className='preview-container'>
+              <span
+                className='remove-img-icon'
+                onClick={() => {
+                  URL.revokeObjectURL(file.preview)
+                  setFile(null)
+                }}
+              >
+                <i className='eos-icons'>delete</i>
+              </span>
+              <img
+                src={file.preview}
+                className='user-profile-avatar-img'
+                alt='preview'
+              />
+            </aside>
+          ))}
+        {error && <FormError message={error} />}
```

Comment:
```suggestion
        {!!error && <FormError message={error} />}
```
error is of type string, if it has any value, react will display this and the component both. convert it to type bool by using `!!`